build.sbt

```diff
@@ -8,25 +8,27 @@ crossPaths := false
 
 publishMavenStyle := true
 
-publishTo <<= version { (v: String) =>
+publishTo := {
   val nexus = "https://oss.sonatype.org/"
-  if (v.trim.endsWith("SNAPSHOT")) Some("snapshots" at nexus + "content/repositories/snapshots")
-  else Some("releases" at nexus + "service/local/staging/deploy/maven2")
+  if (isSnapshot.value)
```

Comment:
sbt 0.13.15 no longer allowed the `<<=` syntax I replaced the code with the example from [sbt docs > sonatype > maven](http://www.scala-sbt.org/0.13/docs/Using-Sonatype.html#Maven+configuration+tips%E2%80%99n%E2%80%99tricks)